vllm/v1/worker/gpu_model_runner.py

```diff
@@ -222,6 +223,12 @@ def __init__(
                 device="cpu",
                 pin_memory=self.pin_memory)
 
+        # Only relevant for models using ALiBi (e.g, MPT)
+        self.use_alibi = (hasattr(model_config, "hf_config")
+                          and hasattr(model_config.hf_config, "attn_config")
+                          and model_config.hf_config.attn_config.get(
+                              "alibi", False))
```

Comment:
I think `model_config` always has `hf_text_config` (defaults to `hf_config` if the model is text only).
```suggestion
        self.use_alibi = (hasattr(model_config.hf_text_config, "attn_config")
                                    and model_config.hf_text_config.attn_config.get("alibi", False))
```

Reply:
Does it work for other ALiBi models besides MPT?